crates/forge/bin/cmd/test/filter.rs

```diff
@@ -38,6 +38,10 @@ pub struct FilterArgs {
         value_name = "GLOB"
     )]
     pub path_pattern_inverse: Option<GlobMatcher>,
+
+    /// Only show coverage for files that do not match the specified regex pattern.
+    #[arg(long = "no-match-coverage", visible_alias = "nmco", value_name = "REGEX")]
+    pub coverage_pattern_inverse: Option<regex::Regex>,
```

Comment:
I've opted for `no-match-coverage` to stay in line with the `no-match-*` pattern established (`no-match-test`, `no-match-contract`, `no-match-path`) 

See: https://github.com/foundry-rs/foundry/pull/7301#discussion_r1521015253